components/climber.py

```diff
@@ -1,11 +1,11 @@
-from magicbot import tunable, will_reset_to
+from magicbot import tunable
 from rev import SparkMax, SparkMaxConfig
 
 from ids import SparkId
 
 
 class ClimberComponent:
-    target_speed = will_reset_to(0.0)
+    target_speed = 0.0
```

Comment:
will_reset_to didn't work in our testing and apparently we've identified the issue before

Reply:
@auscompgeek can you comment on this?

@65-7a what do you mean didn't work?

I don't know that I expect the motor to stop immediately with the current command set because 0 voltage is not the same as stop the motor

Reply:
`will_reset_to` doesn't have an effect in test mode. Maybe that's what they're referring to?

Reply:
@LucienMorey even though we [set the SPARK MAX to brake mode](https://github.com/thedropbears/pyreefscape/blob/f3049f2a327b473c865f65150e6d2d37cd940e30/components/climber.py#L18)?

@auscompgeek oh 💀

Reply:
Brake mode doesn't change the fact that inertia is a fundamental law of physics. It just uses back EMF to slow down and resist motion.

Reply:
Why doesnt it run in test mode? That seems like an oversight. What are the scenarios in which you don't want that thing to reset in test but do in teleop?

Reply:
🤷‍♂️